app/javascript/controllers/federation_controller.js

```diff
@@ -0,0 +1,46 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="federation"
+export default class extends Controller {
+    static values = {
+        config: Object,
+      }
+    static targets = ['chips']
+    connect() {
+      this.#initIconsStyle()
+      this.#initInputChips()
+    }
+    #initIconsStyle(){
+      const style = document.createElement('style');
```

Comment:
Can't you do this in Ruby when rendering the results? adding <style></style> after each render.

Reply:
It didn't work for me

Reply:
OK